Examples/BookingSystem.AspNetCore/Stores/FacilityStore.cs

```diff
@@ -325,20 +325,15 @@ protected async override Task GetOrderItems(List<OrderItemContext<FacilityOpport
                                                      InScheme = new Uri("https://openactive.io/activity-list")
                                                  }
                                              },
-                                         },
-                                         StartDate = (DateTimeOffset)slot.Start,
-                                         EndDate = (DateTimeOffset)slot.End,
-                                         MaximumUses = slot.MaximumUses,
-                                         // Exclude current Order from the returned lease count
-                                         RemainingUses = slot.RemainingUses - db.Count<OrderItemsTable>(
```

Comment:
Make this into another db call that returns number of remaining uses minus leases

Reply:
Done